docker-compose.yml

```diff
@@ -13,6 +13,7 @@ services:
             context: .
             target: dev
         volumes:
+            - dist:/var/www/html/public/app/themes/justice/dist
```

Comment:
Why does this need access to this directory?

Reply:
As part of the js build: laravel mix + @tinypixelco/laravel-mix-wp-blocks will create n asset file e.g. `dist/block-editor.asset.php` it contains dependencies and a version number. These are then used is then used in `wp_register_script`.

In docker-compose, this is above the `.:/var/www/html` because the order of them makes a difference.